<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Symbols</family>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 238, 151);
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ffc400, stop: 1 #ffff00);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 161, 194);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b831c4, stop: 1 #c43169);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_bracket">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_bracket">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(255, 210, 161);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff7700, stop: 1 #ffb700);
}

background-color: rgb(255, 165, 76);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>-1</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit_calculate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color:#ff7700;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #ff0000, stop: 1 #ff7300);
}</string>
    </property>
    <property name="text">
     <string>Credit calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graphic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(101, 198, 255);
	color: black;
	background-color: rgb(219, 255, 179);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1:0, x2: 0, y2: 1, stop: 0 #b4f564, stop: 1 #69f564);
}</string>
    </property>
    <property name="text">
     <string>Graphic</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Value =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_value">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>453</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_x_start">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_y_start">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_y_end">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_x_end">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: white;
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
